packages/common/src/tools/Logger.ts

```diff
@@ -83,6 +88,9 @@ export class Logger {
   }
 
   private print(level: string, parameters: LoggerParameters) {
+    if (this.loggerHistory.includes(parameters)) return
```

Comment:
i think we need a more complicated criterion for skipping logs here. 

It should work so that if there is a repeated log for x times in y seconds, we throttle it for z seconds. Furthermore, once we are done with throttling it would be amazing to write a summary like "message A was throttled b times during last z seconds". @sz-piotr thoughts? If this should be better defined i can write a longer explanation in the ticket.